jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -98,33 +98,35 @@ pipeline {
                         echo "Add Component: ${local_component}"
                         componentTests["Run Integtest ${local_component}"] = {
                             // Using scripted pipelines to trigger dynamic parallel stages
-                            node(AGENT_LABEL) {
-                                docker.image(DOCKER_AGENT.image).inside(DOCKER_AGENT.args) {
-                                    try {
-                                        stage("Run Integtest ${local_component}") {
-                                            echo "Component Name: ${local_component}"
-                                            unstash 'opensearch-build-repo'
-                                            // Jenkins tend to not clean up workspace at times even though ws clean is called
-                                            // Due to docker is mounting the agent directory so it can communicated with the agent
-                                            // This sometimes causes the workspace to retain last run test-results and ends with build failures
-                                            // https://github.com/opensearch-project/opensearch-build/blob/6ed1ce3c583233eae4fe1027969d778cfc7660f7/src/test_workflow/test_recorder/test_recorder.py#L99
-                                            sh("rm -rf test-results")
-                                            runIntegTestScript(
-                                                jobName: BUILD_JOB_NAME,
-                                                componentName: "${local_component}",
-                                                buildManifest: BUILD_MANIFEST,
-                                                testManifest: "manifests/${TEST_MANIFEST}"
+                            timeout(time: 2, unit: 'HOURS') {
```

Comment:
Since we are adding 2hrs timeout here, do we need to extend the timeout for this job accordingly?

Reply:
The job itself is having a 3HR timeout, each parallel stage has 2HR timeout, I dont see any issues with this setup.
At maximum all tests will complete in parallel in 2 hours, and rest of the parts will complete in 1 hour.